src/components/ErrorToast.js

```diff
@@ -10,6 +11,10 @@ const ErrorToast = ({ message }) => {
   );
 };
 
+ErrorToast.propTypes = {
```

Comment:
Be consistent -- propTypes before defaultProps or vice versa, not one or the other.

Reply:
Fixed